### PR TITLE
fix[style]: Refacto display of forms centreon web

### PR DESCRIPTION
**Pull request by emabassi-ext**
## Description

Refacto the display of all configuration/paremeter form.

**Fixes** # MON-16111
## Type of change

- [x] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software

## Target serie

- [ ] 21.10.x
- [ ] 22.04.x
- [ ] 22.10.x
- [x] 23.04.x (master)

<h2> How this pull request can be tested ? </h2>

Labels must take maximum 33 characters spaces included . 
 
check on the pages with the longest labels the impact on the height of the lines

 space  32 pixel between labels and fields. 

#### Community contributors & Centreon team

- [x] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [x] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

File: centreon/www/Themes/Generic-theme/style.css [hunk 1]
```diff
@@ -1210,7 +1210,8 @@ table-form-header-h3 {
 }
 
 .table .FormRowField {
-    padding: 0 6px;
+    width: 206px;
```
Comment (Thebarda) [inline centreon/www/Themes/Generic-theme/style.css:1213]: Use responsive units such as "%" or "vw"

**Review by Wedierre — CHANGES_REQUESTED** [review]
Be careful to align the beginning of sentences, for all sentences that are on several lines.
see on the screenshot for exemple
Capture d’écran 2023-01-24 à 13 39 08

**Comment by Wedierre** [comment]
> but when there is a tooltips before texte the new line starts with the circled question mark

Align text with the beginning of the sentence, not the pictogram
Capture d’écran 2023-01-27 à 14 06 57

**Review by Wedierre — COMMENTED** [review]
on all screen copies there is a problem of centering the text and pictorial on the height of the lines.
Capture d’écran 2023-02-07 à 09 54 07
Capture d’écran 2023-02-07 à 10 02 14
Capture d’écran 2023-02-07 à 10 01 46
Capture d’écran 2023-02-07 à 10 00 33
Capture d’écran 2023-02-07 à 09 59 47
Capture d’écran 2023-02-07 à 09 59 11
Capture d’écran 2023-02-07 à 09 58 12
Capture d’écran 2023-02-07 à 09 57 34
Capture d’écran 2023-02-07 à 09 55 44
Capture d’écran 2023-02-07 à 09 52 44
Capture d’écran 2023-02-07 à 09 29 04
Capture d’écran 2023-02-07 à 09 21 38

Can you fix it easily?

**Review by bdauria — APPROVED** [review]

**Review by Wedierre — APPROVED** [review]

**Review by kduret — APPROVED** [review]

**Review by jeremyjaouen — APPROVED** [review]

**Pull request merged**